static/js/subs.js

```diff
@@ -164,6 +167,13 @@ exports.update_stream_description = function (sub, description, rendered_descrip
 
     // Update stream settings
     stream_edit.update_stream_description(sub);
+
+    // Update navbar stream description
+    const stream_description = $(".narrow_description");
+    if (stream_description !== undefined) {
+        stream_description.html("");
+        stream_description.append(sub.rendered_description);
```

Comment:
This commit could call into a similar no-op function in `tab_bar.js`.  

Also, would it be better to do `stream_description.html(sub.rendered_description)` rather than clearing and then re-adding?